src/main/kotlin/org/rust/lang/core/completion/RsLintCompletionProvider.kt

```diff
@@ -0,0 +1,545 @@
+        Lint.Clippy("type_complexity"),
+        Lint.Clippy("type_repetition_in_bounds"),
+        Lint.Clippy("unicode_not_nfc"),
+        Lint.Clippy("unimplemented"),
+        Lint.Clippy("uninit_assumed_init"),
+        Lint.Clippy("unit_arg"),
+        Lint.Clippy("unit_cmp"),
+        Lint.Clippy("unknown_clippy_lints"),
+        Lint.Clippy("unnecessary_cast"),
+        Lint.Clippy("unnecessary_filter_map"),
+        Lint.Clippy("unnecessary_fold"),
+        Lint.Clippy("unnecessary_mut_passed"),
+        Lint.Clippy("unnecessary_operation"),
+        Lint.Clippy("unnecessary_sort_by"),
+        Lint.Clippy("unnecessary_unwrap"),
+        Lint.Clippy("unneeded_field_pattern"),
+        Lint.Clippy("unneeded_wildcard_pattern"),
+        Lint.Clippy("unnested_or_patterns"),
+        Lint.Clippy("unreachable"),
+        Lint.Clippy("unreadable_literal"),
+        Lint.Clippy("unsafe_derive_deserialize"),
+        Lint.Clippy("unsafe_removed_from_name"),
+        Lint.Clippy("unsafe_vector_initialization"),
+        Lint.Clippy("unseparated_literal_suffix"),
+        Lint.Clippy("unsound_collection_transmute"),
+        Lint.Clippy("unstable_as_mut_slice"),
+        Lint.Clippy("unstable_as_slice"),
+        Lint.Clippy("unused_collect"),
+        Lint.Clippy("unused_io_amount"),
+        Lint.Clippy("unused_label"),
+        Lint.Clippy("unused_self"),
+        Lint.Clippy("unused_unit"),
+        Lint.Clippy("unwrap_used"),
+        Lint.Clippy("use_debug"),
+        Lint.Clippy("use_self"),
+        Lint.Clippy("used_underscore_binding"),
+        Lint.Clippy("useless_asref"),
+        Lint.Clippy("useless_attribute"),
+        Lint.Clippy("useless_conversion"),
+        Lint.Clippy("useless_format"),
+        Lint.Clippy("useless_let_if_seq"),
+        Lint.Clippy("useless_transmute"),
+        Lint.Clippy("useless_vec"),
+        Lint.Clippy("vec_box"),
+        Lint.Clippy("vec_resize_to_zero"),
+        Lint.Clippy("verbose_bit_mask"),
+        Lint.Clippy("verbose_file_reads"),
+        Lint.Clippy("vtable_address_comparisons"),
+        Lint.Clippy("while_immutable_condition"),
+        Lint.Clippy("while_let_loop"),
+        Lint.Clippy("while_let_on_iterator"),
+        Lint.Clippy("wildcard_dependencies"),
+        Lint.Clippy("wildcard_enum_match_arm"),
+        Lint.Clippy("wildcard_imports"),
+        Lint.Clippy("wildcard_in_or_patterns"),
+        Lint.Clippy("write_literal"),
+        Lint.Clippy("write_with_newline"),
+        Lint.Clippy("writeln_empty_string"),
+        Lint.Clippy("wrong_pub_self_convention"),
+        Lint.Clippy("wrong_self_convention"),
+        Lint.Clippy("wrong_transmute"),
+        Lint.Clippy("zero_divided_by_zero"),
+        Lint.Clippy("zero_prefixed_literal"),
+        Lint.Clippy("zero_ptr"),
+        Lint.Clippy("zero_width_space"),
+        Lint.Clippy("zst_offset")
+    )
+}
+
+private sealed class Lint(val name: String) {
+    class RustcSpecific(name: String) : Lint(name)
```

Comment:
```suggestion
    class Rustc(name: String) : Lint(name)
```
nit: I think `Group` suffix is enough to distinguish them